bluetooth_mesh/mesh.py

```diff
@@ -166,3 +211,30 @@ def pack(self, application_key, network_key, seq, iv_index):
 
             yield seq, bitstring.pack('uint:1, uint:7, bits, bytes',
                                       iv_index & 1, nid, obfuscated_header, network_pdu).bytes
+
+    @classmethod
+    def unpack(cls, payload, network_keys, local_iv_index):     
+        ivi, nid = bitstring.BitString(payload).unpack('uint:1, uint:7')
+        keys = [key.encryption_keys for key in network_keys if key.encryption_keys[0] == nid]
+        if keys:
+            _, encryption_key, privacy_key = keys[0]
+            obfuscated = payload[1:7]
+            iv_index = local_iv_index if ivi == 0 else local_iv_index - 1
```

Comment:
I don't think this is correct.

As far as I understand the spec, IVI is the least significant bit of the IV Index, so for local_iv_index = 0x11 and incoming ivi = 0, iv_index should be 0x10.

Reply:
I guess the description of IVI is wrong in the spec. 

in chapter **3.10.5**:
_During the IV Update in Progress state, the IV Update Flag in the Secure Network beacon and in the Friend Update message shall be set to 1. When this state is active, a node shall transmit using the current IV Index - 1 and shall process messages from the current IV Index - 1 and also the current IV Index._

so I think IVI is independent of IV Index